src/main/java/com/hubspot/jinjava/objects/collections/PyList.java

```diff
@@ -28,6 +29,10 @@ public boolean append(Object e) {
   }
 
   public void insert(int i, Object e) {
+    if (i < 0 || i >= list.size()) {
```

Comment:
Just checking that we don't support negative indexing as in Python? From testing in a Jinja interpreter this seems to be supported.